app/src/main/res/values/static_strings.xml

```diff
@@ -60,4 +60,44 @@
     <!-- Secret debug info strings -->
     <string name="debug_info_region_home" translatable="false">Home region</string>
     <string name="debug_info_region_current" translatable="false">Current region</string>
+
+    <!-- Profiler settings -->
+    <string name="preferences_start_profiler">Start Profiler</string>
+    <string name="profiler_stop">Stop Profiler</string>
+    <string name="profiler_settings_title">Profiler Settings</string>
+    <string name="profiler_filter_firefox">Firefox</string>
+    <string name="profiler_running">Profiler is currently running</string>
+    <string name="profiler_filter_firefox_explain">Recommended preset for profiling Firefox</string>
+    <string name="profiler_filter_graphics">Graphics</string>
+    <string name="profiler_filter_graphics_explain">Preset for investigating graphics bugs in Firefox</string>
+    <string name="profiler_filter_media">Media</string>
+    <string name="profiler_filter_media_explain">Preset for investigating audio and video bugs in Firefox</string>
+    <string name="profiler_filter_networking">Networking</string>
+    <string name="profiler_filter_networking_explain">Preset for investigating networking bugs in Firefox</string>
+    <string name="profiler_start_dialog_started">Profiler started</string>
+    <string name="profiler_save_method">Would you like to save your profile as:</string>
+    <string name="profiler_save_method_url">URL</string>
+    <string name="profiler_save_method_url_explained">The profile will upload automatically with all the information gathered and provide you a link from profiler.firefox.com</string>
+    <string name="profiler_save_method_file">File</string>
+    <string name="profiler_save_method_file_explained">The file will be saved locally on your android device. To share it, go to profiler.firefox.com on your computer or mobile device</string>
+    <string name="profiler_start_cancel">Cancel</string>
+    <string name="profiler_gathering">Gathering the profile</string>
+    <string name="profiler_waiting_start">Waiting for Profiler to start</string>
+    <string name="profiler_url_warning">Saving to URL warning</string>
+    <string name="profiler_url_warning_explained">"Saving to URL could potentially show private information such as " +
+                        "urls and screenshots. If you want to keep these" +
+                        "privates, save your profile as a file instead and head on over to profiler.firefox.com" +
+                        "where you can select what to share and not share."</string>
```

Comment:
This needs to be updated from the previous version. Also, I think it should be more formal

```suggestion
    <string name="profiler_url_warning_explained">Performance profiles may contain sensitive information such as websites visited and screenshots of them. If you continue, this recent profile will be uploaded publicly. Do you wish to continue?</string>
```